sqlflow/client.py

```diff
@@ -8,9 +9,10 @@
 import sqlflow.proto.sqlflow_pb2 as pb
 import sqlflow.proto.sqlflow_pb2_grpc as pb_grpc
 
-
 _LOGGER = logging.getLogger(__name__)
-
+handler = logging.StreamHandler(sys.stdout)
+_LOGGER.setLevel(logging.INFO)
+_LOGGER.addHandler(handler)
```

Comment:
I am not sure adding the log handler here is a good idea. Will those lines be executed every time we import the package?

@zou000 @ktong What do you think?

Reply:
Seems the Python interpreter would use a global table to cache the imported module, for the `import module` statement:
> if module is not imported python imports it, creates module object and puts newly created module object to the global table
if module is imported python just gets module object

ref: https://stackoverflow.com/questions/37067414/python-import-multiple-times/37077911

Reply:
You may clean all logging handlers before adding a new one every time to ensure that there are always only one handler global

Reply:
@typhoonzero  This would lead to re-initialize the logger when reloading the sqlflow magic command, seems it's not necessary?